src-ui/cypress/integration/documents-list.spec.ts

```diff
@@ -0,0 +1,20 @@
+describe('documents-list', () => {
+  beforeEach(() => {
+    cy.intercept('http://localhost:8000/api/documents/*', {
+      fixture: 'documents/documents.json',
+    });
+    cy.intercept('http://localhost:8000/api/documents/1/thumb/', {
+      fixture: 'documents/lorem-ipsum.png',
+    });
+
+    cy.visit('/documents');
```

Comment:
I think `cy.visit` should be moved to the test, so that this spec is resilient to future modifications / refactorings?

Reply:
yes, agree